iroh-dns-server/Cargo.toml

```diff
@@ -24,7 +24,7 @@ derive_more = { version = "1.0.0", features = [
     "from",
 ] }
 dirs-next = "2.0.0"
-governor = "0.6.3" #needs new release of tower_governor for 0.7.0
+governor = "0.8"
```

Comment:
Bonus points if we can move `iroh-relay` to govenor `0.8`, too, so we don't depend on govenor in two different versions here.
Although admittedly, this is only for server binaries, so doesn't matter that much.

Reply:
done